src/storage/mvcc/reader/reader.rs

```diff
@@ -1602,7 +1636,7 @@ mod tests {
         let mut reader = MvccReader::new(snap, None, false, IsolationLevel::Si);
         for &skip_lock_check in &[false, true] {
             assert_eq!(
-                reader.get(&key, 11.into(), skip_lock_check).unwrap(),
+                reader.get(&key, 11.into(), None, skip_lock_check).unwrap(),
```

Comment:
Should there be some tests with `gc_fence_limit` not `None`?

Reply:
It should be covered by tests of  the `get`'s invokers. It's good also to test it but I'm afraid that this single PR is too large...